app/src/main/java/org/sopt/pingle/presentation/ui/newgroup/NewGroupActivity.kt

```diff
@@ -106,21 +143,36 @@ class NewGroupActivity : BindingActivity<ActivityNewGroupBinding>(R.layout.activ
         }
     }
 
-    private fun navigateToNewGroupAnnouncement() {
+    private fun navigateToNewGroupAnnouncement(newGroupModel: NewGroupModel) {
         Intent(this, NewGroupAnnouncementActivity::class.java).apply {
+            putExtra(NEW_GROUP_CODE, newGroupModel)
             startActivity(this)
         }
+        finish()
     }
 
     private fun navigateToPreviousPage() {
         when (binding.vpNewGroup.currentItem) {
             FIRST_PAGE -> finish()
+
             else -> binding.vpNewGroup.currentItem--
         }
     }
 
+    private fun onBackPressedBtn() {
+        onBackPressed = object : OnBackPressedCallback(true) {
+            override fun handleOnBackPressed() {
+                navigateToPreviousPage()
+            }
+        }
+        onBackPressedDispatcher.addCallback(this, onBackPressed)
+    }
```

Comment:
```suggestion
        onBackPressedDispatcher.addCallback(this, object : OnBackPressedCallback(true) {
            override fun handleOnBackPressed() {
                navigateToPreviousPage()
            }
        })
```

이런 식으로 쓸 수 있는 것도 아시나요?

Reply:
오오오!? 이런식으로 작성하는게 더 깔끔해보이네요!